demisto_sdk/commands/validate/file_validator.py

```diff
@@ -101,16 +101,22 @@ def __init__(self, is_backward_check=True, prev_ver=None, use_git=False, is_circ
         self._is_valid = True
         self.configuration = configuration
         self.is_backward_check = is_backward_check
-        self.is_circle = is_circle
+        self.is_circle = only_committed_files
         self.print_ignored_files = print_ignored_files
-        self.validate_conf_json = validate_conf_json
+        self.skip_conf_json = skip_conf_json
         self.validate_id_set = validate_id_set
         self.file_path = file_path
         self.skip_pack_rn_validation = skip_pack_rn_validation
         self.changed_pack_data = set()
 
-        if self.validate_conf_json:
+        self.is_private_repo = is_private_repo
+        if is_private_repo:
+            print('Running in a private repository')
+            self.skip_conf_json = True  # private repository don't have conf.json file
```

Comment:
This is their private repository, they can decide what to run, how to run it.
The partner will have to figure out how to run test playbooks  (probably it will not be part of the build - my guess). So there is no need to have `conf.json`